doc/juju-core-release-process.txt

```diff
@@ -58,8 +58,8 @@ version/version.go
 
 propose it and commit it. This moves the development version to the next release number in the series.
 
-Step 6. Upload tools to the s3 public bucket
-============================================
+Step 6. Upload agent binaries to the s3 public bucket
```

Comment:
I wonder if any of this document is still correct.
We should ask balloons/veebers to review.

Reply:
asked via email but am hoping to land this first before any conflicts arise ;D